lua/wikis/hearthstone/PrizePool/Custom.lua

```diff
@@ -0,0 +1,68 @@
+---
+-- @Liquipedia
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local Arguments = Lua.import('Module:Arguments')
+local Class = Lua.import('Module:Class')
+local Logic = Lua.import('Module:Logic')
+local Variables = Lua.import('Module:Variables')
+
+local PrizePool = Lua.import('Module:PrizePool')
+local Opponent = Lua.import('Module:Opponent/Custom')
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector')
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+local TIER_VALUE = {8, 4, 2}
+local TIER_TYPE_MODIFIER = {Showmatch = 0.01, Monthly = 0.4, Weekly = 0.2, Daily = 0.1}
+
+-- Template entry point
+---@param frame Frame
+---@return Html
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	local prizePool = PrizePool(args):create()
+
+	prizePool:setLpdbInjector(CustomLpdbInjector())
+
+	return prizePool:build()
+end
+
+---@param lpdbData placement
+---@param placement PrizePoolPlacement
+---@param opponent BasePlacementOpponent
+---@return placement
+function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
+	lpdbData.weight = CustomPrizePool.calculateWeight(
+		lpdbData.prizemoney,
+		Variables.varDefault('tournament_liquipediatier'),
+		placement.placeStart,
+		Variables.varDefault('tournament_liquipediatiertype')
+	)
+
```

Comment:
you likely want to add the stuff that commons sets here ;)
you can check https://github.com/Liquipedia/Lua-Modules/blob/0e201e4b167899782857a5eda8f5246c0bfefb3f/lua/wikis/commons/PrizePool/Custom.lua